src/main/scala/nl.knaw.dans.easy.solr4files/components/DDM.scala

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.solr4files.components
+
+import java.net.URL
+
+import nl.knaw.dans.easy.solr4files.SolrLiterals
+import nl.knaw.dans.easy.solr4files.components.DDM._
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+
+import scala.xml.{ Node, XML }
+
+class DDM(xml: Node) extends DebugEnhancedLogging {
+
+  val accessRights: String = (xml \ "profile" \ "accessRights").text
```

Comment:
given that you do a lot of `xml \ "profile"` and `xml \ "dcmiMetadata"`, why don't you put them in values first?